src/main/java/winstone/Launcher.java

```diff
@@ -179,11 +178,6 @@ public Launcher(Map<String, String> args) throws IOException {
                 server.addBean(mbeanContainer);
             }
 
-            // JENKINS-73616: Turn down log level of annotation parser
-            java.util.logging.Logger logger =
-                    java.util.logging.Logger.getLogger("org.eclipse.jetty.ee9.annotations.AnnotationParser");
-            logger.setLevel(Level.SEVERE);
-
```

Comment:
Could no longer reproduce [JENKINS-73616](https://issues.jenkins.io/browse/JENKINS-73616) after excluding the `jetty-ee10-annotations` package in `pom.xml`, so removing this workaround.